rsconnect/api.py

```diff
@@ -37,6 +45,21 @@ def __init__(self, url, api_key, insecure=False, ca_data=None):
         # This is specifically not None.
         self.cookie_jar = CookieJar()
 
+    def handle_bad_response_generic(self, response):
```

Comment:
This is no longer needed. `handle_bad_response()` now handles this case on the main branch